Automattic/wp-calypso
# FSE: Add API endpoint to side-load images

#### Changes proposed in this Pull Request

* Adds the ability to side-load images. 

#### Testing instructions

Send a POST request to the `/wp-json/wp/v2/media/image` endpoint on your test site with an image url in the body. 

See  #34603
